### PR TITLE
fix release notes

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran.alex@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by hfxsd — APPROVED** [review]
LGTM

File: releases/release-6.1.0.md [hunk 1]
```diff
@@ -408,20 +405,20 @@ TiDB 版本：6.1.0
     + TiDB Data Migration (DM)
 
         - 修复 `start-time` 时区问题，从使用下游时区改为使用上游时区  [#5471](https://github.com/pingcap/tiflow/issues/5471)
-        - 修复任务自动恢复后，DM 会占用更多磁盘空间的问题[#3734](https://github.com/pingcap/tiflow/issues/3734), [#5344](https://github.com/pingcap/tiflow/issues/5344)
-        - 修复 checkpoint flush 可能导致失败行数据被跳过的问题[#5279](https://github.com/pingcap/tiflow/issues/5279)
+        - 修复任务自动恢复后，DM 会占用更多磁盘空间的问题 [#3734](https://github.com/pingcap/tiflow/issues/3734), [#5344](https://github.com/pingcap/tiflow/issues/5344)
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:408]:
```suggestion
        - 修复任务自动恢复后，DM 会占用更多磁盘空间的问题 [#3734](https://github.com/pingcap/tiflow/issues/3734)，[#5344](https://github.com/pingcap/tiflow/issues/5344)
```

File: releases/release-6.1.0.md [hunk 2]
```diff
@@ -376,7 +373,7 @@ TiDB 版本：6.1.0
 
     - 修复下线一个 TiKV 实例导致 Raft log lag 越来越大的问题 [#12161](https://github.com/tikv/tikv/issues/12161)
     - 修复待 merge 的 Region 无效会导致 TiKV panic 且非预期地销毁 peer 的问题 [#12232](https://github.com/tikv/tikv/issues/12232)
-    - 修复从 v5.3.1、v5.4.0 升级到 v6.0.0 时 TiKV 报 `failed to load_latest_options` 错误的问题 [#12269](https://github.com/tikv/tikv/issues/12269)
+    - 修复从 v5.3.1、v5.4.0 升级到 v6.0.0 及以上版本时 TiKV 报 `failed to load_latest_options` 错误的问题 [#12269](https://github.com/tikv/tikv/issues/12269)
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:376]: L386
    ``- 修复已清除的 `tombstone store` 信息在切换 PD leader 后再次出现的问题 [#4941](https://github.com/tikv/pd/issues/4941)``

File: releases/release-6.1.0.md [hunk 3]
```diff
@@ -138,20 +133,20 @@ TiDB 版本：6.1.0
...
+  对于从 v6.1.0 之前版本升级到 v6.1.0 的集群（包括滚动升级和停机升级），请注意：
 
     * 若升级前集群指定的配置文件中，存在已经配置的项，则升级过程中 TiDB 将会将配置项的值自动更新为对应系统变量的值，以保证升级后，系统的行为不会因为参数的优化发生变化。
     * 上述自动更新仅在升级过程中发生一次，升级完成之后，被废弃的配置项不再有任何效果。
 
-    通过本次优化，客户可以在线修改参数生效并持久化，方便客户进行日常运维，避免重启生效对在线业务造成的影响。
+  通过本次优化，客户可以在线修改参数生效并持久化，方便客户进行日常运维，避免重启生效对在线业务造成的影响。
 
-    [用户文档](/dynamic-config.md)
+  [用户文档](/dynamic-config.md)
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:149]: L155 ``当开启 `enable-global-kill` 配置项时``

File: releases/release-6.1.0.md [hunk 4]
```diff
@@ -408,20 +405,20 @@ TiDB 版本：6.1.0
     + TiDB Data Migration (DM)
 
         - 修复 `start-time` 时区问题，从使用下游时区改为使用上游时区  [#5471](https://github.com/pingcap/tiflow/issues/5471)
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:407]:
```suggestion
        - 修复 `start-time` 时区问题，从使用下游时区改为使用上游时区 [#5471](https://github.com/pingcap/tiflow/issues/5471)
```

**Review by shichun-0415 — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**